pkgs/servers/photoprism/default.nix

```diff
@@ -0,0 +1,86 @@
+{ pkgs, lib, stdenv, fetchFromGitHub, fetchzip, darktable, rawtherapee, ffmpeg, libheif, exiftool, nixosTests, makeWrapper }:
+
+let
+  version = "221102-905925b4d";
```

Comment:
```suggestion
  version = "unstable-2022-11-02";
```

Reply:
I think that's just their versioning scheme. According to the phrasing in https://github.com/photoprism/photoprism/blob/7a295cab4931d15d685e272b9363c734cfe78c0f/README.md?plain=1#L47, those releases are considered stable. Should I contact the developer to make sure?

Reply:
Ah ok, fair enough them.